build.rs

```diff
@@ -0,0 +1,14 @@
+use cfgrammar::yacc::YaccKind;
+use lrlex::CTLexerBuilder;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    CTLexerBuilder::new()
+        .lrpar_config(|ctp| {
+            ctp.yacckind(YaccKind::Grmtools)
+                .grammar_in_src_dir("calc.y")
+                .unwrap()
```

Comment:
Since the parent function returns `Result<_, Box<dyn Error>`, you should be able to bubble any error from `grammar_in_src_dir()` upwards using the `?` operator.

Try:
```
.grammar_in_src_dir("calc.y")?
```

Reply:
(and the unwrap should then not be needed)

Reply:
ok, will add error propagation :muscle: 
this code was copied almost as-is from the quickstart

Reply:
FWIW `unwrap` in `build.rs` is normally fine IMHO, since it only affects those *compiling* the program not those *running* the program.

Reply:
[Also, `unwrap` prints out the line number of the error, whereas `?` doesn't. So I personally tend to prefer `unwrap` in `build.rs`. It's not a big deal either way, though.]

Reply:
Oh, it's a `build.rs`. yeah ignore me then.